.github/workflows/test-docs.yml

```diff
@@ -0,0 +1,51 @@
+name: Build API Doc
+
+on: [workflow_dispatch]
+
+jobs:
+  api:
+    name: Deploy API Documentation
+    runs-on: ubuntu-latest
+    env:
+      root: https://github.com/sfarrens/
```

Comment:
This should rather be targeting to CEA-COSMIC I guess

Reply:
Ahhh now maybe yes, initially, it was not yet up